data-reconciliation-app/tsconfig.json

```diff
@@ -3,12 +3,14 @@
     "lib": ["ES2020"],
     "target": "ES2020",
     "module": "ES2020",
+    "esModuleInterop": true,
```

Comment:
I suggest following `tsconfig` for maximum backwards compatibility and developer productivity:

```json
{
    "compilerOptions": {
        "target": "ES2018",
        "module": "commonjs",
        "lib": [
            "es2018"
        ],
        "declaration": true,
        "resolveJsonModule": true,
        "strict": true,
        "noImplicitAny": false,
        "strictNullChecks": true,
        "noImplicitThis": true,
        "alwaysStrict": true,
        "noUnusedLocals": false,
        "noUnusedParameters": false,
        "noImplicitReturns": true,
        "noFallthroughCasesInSwitch": false,
        "inlineSourceMap": false,
        "inlineSources": false,
        "experimentalDecorators": true,
        "strictPropertyInitialization": false,
        "esModuleInterop": true,
        "skipLibCheck": true,
        "sourceMap": true,
        "outDir": "build",
        "typeRoots": [
            "./node_modules/@types"
        ],
        "types": [
            "node",
            "jasmine",
            "mocha",
            "jest"
        ]
    },
    "include": [
        "**/*"
    ],
    "exclude": [
        "./build/**/*",
        "docs"
    ]
}
```

Reply:
It causes some issues, I will do it as seperate PR